--- conary-2.5.5/scripts/Makefile	2014-11-05 11:04:58.000000000 -0500
+++ conary-2.5.5/scripts/Makefile.new	2015-05-20 09:34:33.195083922 -0400
@@ -19,7 +19,7 @@ extra_dist = Makefile \
              logaction.py changemail.py \
 	     perlreqs.pl findmissingbuildreqs
 
-bin_scripts = rpm2cpio dbsh conary-debug ccs2tar
+bin_scripts = dbsh conary-debug ccs2tar
 util_scripts = \
 	dumpcontainer \
 	localupdateinfo \
